frontend/cypress/e2e/exercise.cy.ts

```diff
@@ -0,0 +1,826 @@
+            .its('position')
+            .its('transfer')
+            .should('have.property', 'isPaused', true);
+
+        cy.getState()
+            .its('exerciseState')
+            .its('vehicles')
+            .itsValues()
+            .firstElement()
+            .its('position')
+            .its('transfer')
+            .its('endTimeStamp')
+            .then((previousEndTimeStamp: any) => {
+                cy.get('[data-cy=transferTimeReduceByOneMinuteButton]').click();
+
+                cy.get('@trainerSocketPerformedActions')
+                    .lastElement()
+                    .should(
+                        'have.property',
+                        'type',
+                        '[Transfer] Edit transfer'
+                    );
+
+                cy.getState()
+                    .its('exerciseState')
+                    .its('vehicles')
+                    .itsValues()
+                    .firstElement()
+                    .its('position')
+                    .its('transfer')
+                    .its('endTimeStamp')
+                    .should('be.lessThan', previousEndTimeStamp);
+            });
+
+        cy.get('[data-cy=transferTimeTogglePauseButton]').click();
+
+        cy.get('@trainerSocketPerformedActions')
+            .lastElement()
+            .should(
+                'have.property',
+                'type',
+                '[Transfer] Toggle pause transfer'
+            );
+
+        cy.getState()
+            .its('exerciseState')
+            .its('vehicles')
+            .itsValues()
+            .firstElement()
+            .its('position')
+            .its('transfer')
+            .should('have.property', 'isPaused', false);
+
+        cy.get('[data-cy=transferOverviewCloseButton]').click();
+        cy.get('[data-cy=trainerToolbarStartButton]').click();
+        cy.get('[data-cy=confirmationModalOkButton]').click();
+
+        cy.wait(1000);
+        cy.getState()
+            .its('exerciseState')
+            .its('vehicles')
+            .itsValues()
+            .firstElement()
+            .its('position')
+            .should('have.property', 'type', 'coordinates');
+    });
+
+    it('can start and stop an exercise', () => {
+        cy.log('start an exercise')
+            .get('[data-cy=trainerToolbarStartButton]')
+            .click();
+        cy.get('[data-cy=confirmationModalOkButton]').click();
+
+        cy.wait(commonErrorTimeout);
+
+        cy.get('@trainerSocketPerformedActions')
+            .atPosition(-2)
+            .should('have.property', 'type', '[Exercise] Start');
+
+        cy.get('@trainerSocketPerformedActions')
+            .lastElement()
+            .should(
+                'have.property',
+                'type',
+                '[Emergency Operation Center] Add Log Entry'
+            );
+
+        cy.getState()
+            .its('exerciseState')
+            .should('have.property', 'currentStatus', 'running');
+
+        cy.wait(1000);
```

Comment:
Oh no a magic number maybe introduce a variable luke commonErrorTimeout such as tickDuration for this

Reply:
See f828d327ae552d4dc91bbbcc5989e7b025c9aff4